src/nerf/websocket.gleam

```diff
@@ -1,20 +1,14 @@
-import gleam/http.{Header}
-import gleam/dynamic.{Dynamic}
+import gleam/http.{type Header}
+import gleam/dynamic.{type Dynamic}
 import gleam/result
-import gleam/string_builder.{StringBuilder}
-import gleam/bit_builder.{BitBuilder}
-import nerf/gun.{ConnectionPid, StreamReference}
+import gleam/string_builder.{type StringBuilder}
+import gleam/bit_builder.{type BitBuilder}
+import nerf/gun.{type ConnectionPid, type StreamReference}
 
 pub opaque type Connection {
   Connection(ref: StreamReference, pid: ConnectionPid)
 }
 
-pub type Frame {
-  Close
-  Text(String)
-  Binary(BitString)
-}
```

Comment:
I think this has been mistakenly removed?

Reply:
type Frame exists as `gun.Frame` already. it was not clear if it was intentional to have two separate types with the same name?

Reply:
The API is intentionally designed, the documentation explains the goals.

Reply:
> The API is intentionally designed, the documentation explains the goals.

maybe I'm missing something. Can you point me to the documentation so I can understand this better.

https://hexdocs.pm/nerf/ is a bit sparse

Reply:
They are deliberately different. The nerf module is a low level binding.